libsolidity/parsing/Parser.cpp

```diff
@@ -382,6 +382,36 @@ Declaration::Visibility Parser::parseVisibilitySpecifier()
 	return visibility;
 }
 
+ASTPointer<OverrideSpecifier> Parser::parseOverrideSpecifier()
+{
+	solAssert(m_scanner->currentToken() == Token::Override, "");
+	ASTNodeFactory nodeFactory(*this);
+
+	nodeFactory.markEndPosition();
+	m_scanner->next();
+
+	std::vector<ASTPointer<UserDefinedTypeName>> overrides;
```

Comment:
Perhaps move this to line 389, such that all local variables are declared in the same location?

Reply:
Moved it up.